src/client/AM.PA.MonitoringTool/WindowsActivityTracker/Models/WindowsActivity.cs

```diff
@@ -5,6 +5,7 @@
 
 using System;
 using System.Collections.Generic;
+using ActivityMapper;
```

Comment:
Is this import used?

Reply:
yes. it is used for the `ActivityCategory ` enum in `public ActivityCategory ActivityCategory { get; set; }`